.github/workflows/on-main-push.yaml

```diff
@@ -40,20 +44,47 @@ jobs:
         uses: docker/setup-qemu-action@v2
 
       - name: Set up Docker Buildx
-        id: buildx
         uses: docker/setup-buildx-action@v2
 
       - name: Find current tag version
         run: echo "::set-output name=sha_short::$(git rev-parse --short HEAD)"
         id: tags
 
+      - name: Setup GoReleaser
+        run: make bootstrap-tools
+
+      - name: Build binaries
+        run: make kured-release-snapshot
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          COSIGN_EXPERIMENTAL: 1
+
       - name: Build image
         uses: docker/build-push-action@v3
         with:
           context: .
-          file: cmd/kured/Dockerfile.multi
           platforms: linux/arm64, linux/amd64, linux/arm/v7, linux/arm/v6, linux/386
           push: true
           tags: |
             docker.io/${{ GITHUB.REPOSITORY }}:main-${{ steps.tags.outputs.sha_short }}
             ghcr.io/${{ GITHUB.REPOSITORY }}:main-${{ steps.tags.outputs.sha_short }}
+
+      - name: Generate SBOM
+        run: |
+          .tmp/syft docker.io/${{ GITHUB.REPOSITORY }}:main-${{ steps.tags.outputs.sha_short }} -o spdx | jq --compact-output > kured-docker.sbom
```

Comment:
If we do this after the move to a new org, does it still make sense to have the docker.io command?

Reply:
fixed